### PR TITLE
Testing updates and Fixing rollout timeouts

### PR DESCRIPTION
**Pull request by chrislovecnm**
### Testing updates

- updating test files to get CI working properly
- Bumping Jenkinsfile container version
- fixing headers that were missed

### Various script updates

- Setting context for every call. Not relying on a content that can
change mid stream.
- Fixing linting problems.
- Setting timeout on rollout

### CODE REVIEW

File: scripts/on-prem-create.sh [hunk 1]
```diff
@@ -46,29 +46,32 @@ fi
 
 # roll out the master,client and data manifest in order due to dependencies
 # roll out master nodes related objects
-kubectl --namespace default apply -f "$PROJECT_ROOT"elasticsearch/manifests/es-discovery-svc.yaml
-kubectl --namespace default apply -f "$PROJECT_ROOT"elasticsearch/manifests/es-svc.yaml
-kubectl --namespace default apply -f "$PROJECT_ROOT"elasticsearch/manifests/es-master.yaml
-kubectl --namespace default rollout status -f "$PROJECT_ROOT"elasticsearch/manifests/es-master.yaml
+kubectl --context "$CONTEXT" apply -f "$PROJECT_ROOT"elasticsearch/manifests/es-discovery-svc.yaml
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply scripts/on-prem-create.sh]: @cxhercules I was trying to get these projects to the point of running both creates in ||, which did not work out.  The problem I have with setting context, is that a user can set context in another window and mess up the script.  I have done that before personally.  These rollouts take about 10 min to deploy.  You ok with using `--context`?

File: scripts/cloud-create.sh [hunk 2]
```diff
@@ -29,23 +29,24 @@ PROJECT_ROOT=$(dirname "${BASH_SOURCE[0]}")/../
 DEFAULT_REPO=gcr.io/$(gcloud config get-value project)
 REPO=${CONTAINER_REPO:-$DEFAULT_REPO}
 
+# shellcheck source=k8s.env
 source "$PROJECT_ROOT"k8s.env
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply scripts/cloud-create.sh]: Good catch.  I don't like it either.

File: scripts/on-prem-create.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (cxhercules) [reply scripts/on-prem-create.sh]: Of course it is fine :)

**Comment by chrislovecnm** [comment]
please test this

**Comment by chrislovecnm** [comment]
test this please

File: Jenkinsfile [hunk 3]
```diff
@@ -19,146 +20,116 @@ limitations under the License.
...
-            sh "make terraform"
-        }
-      }
-    }
+       stage('Create') {
+         container(containerName) {
+           // setup the cluster k8s file
+           sh "make configure"
+           // configure docker so that bazel can upload files
+           sh "gcloud auth configure-docker --quiet"
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply Jenkinsfile]: Yes it uses the docket config to authenticate.

**Pull request merged**